_posts/2018-10-01-welcome-to-freeside.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Welcome 2018 Freshers to Freeside!
+author: Starbeamrainbowlabs
+tags:
+- updates
+---
+
+This is an introductory post to Freeside, a loose group of students who use Linux and want to educate themselves (and others!) about alternative open computing.
+
+## How to find us
+Our physical space is a new Lab that has been purpose built - the _Ritche Lab_ - which is located next to LTD on the 3rd floor of the Robert Blackburn building. It contains computers running Fedora Linux for all students to use - simply register an account [here](https://profiles.freeside.co.uk/join) to get your login credentials.
```

Comment:
link to https://freeside.co.uk/2018/04/15/freeside-is-now-open for more info about the room/computers?